spec/models/rent_spec.rb

```diff
@@ -5,19 +5,35 @@
 
   it { is_expected.to validate_presence_of(:start_date) }
   it { is_expected.to validate_presence_of(:end_date) }
+  it { is_expected.to validate_presence_of(:book) }
+  it { is_expected.to validate_presence_of(:user) }
+end
 
+describe Rent do
   describe '#save' do
-    context 'With valid params' do
-      it 'returns the new rent object' do
-        rent.save!
-        expect(rent).to be_present
+    context 'With invalid params' do
+      let(:withoutBookValue) { build(:rent, book: nil) }
+
+      it 'returns invalid book value' do
+        expect(withoutBookValue).to be_invalid
       end
-    end
 
-    let(:emptyRent) { Rent.new }
-    context 'With invalid params' do
-      it 'returns exception' do
-        expect { emptyRent.save! }.to raise_error(ActiveRecord::RecordInvalid)
+      let(:withoutUserValue) { build(:rent, user: nil) }
+
+      it 'returns invalid user value' do
+        expect(withoutUserValue).to be_invalid
+      end
+
+      let(:withoutStartDateValue) { build(:rent, start_date: '') }
```

Comment:
Use snake case and not camel case, `:without_start_date_value`

Reply:
😞 you're right...done!